src/compas_rhino/geometry/brep/brep.py

```diff
@@ -247,6 +248,25 @@ def from_mesh(cls, mesh):
     # Methods
     # ==============================================================================
 
+    def to_meshes(self, u=16, v=16):
```

Comment:
I know it's `to_meshes` and not `to_mesh` but would it maybe make sense to return a single joint mesh? If remember right, `Mesh.CreateFromBrep` returns a mesh for each face of the Brep, is that something a caller should always expect?